### PR TITLE
remove year from copyright

### PR DESCRIPTION
**Pull request by bruno-garcia**
This came up in code reviews now on an exporter for Sentry but also in other OTel repos:

image

/cc @pjanotti

### CODE REVIEW

**Comment by james-bebbington** [comment]
I'm not really sure of the implications of removing the year from this disclaimer, but if its deemed okay, note that you've only updated a small number of source files in this PR (many files reference 2018, 2019, or include an extra comma after the year). If we're going to make this change, we should presumably update everything at once.

**Comment by bruno-garcia** [comment]
Good catch, Thanks I'll update the others

**Comment by bruno-garcia** [comment]
@james-bebbington should be fixed now. Thanks for reviewing.

/cc @reyang @SergeyKanzhelev who are tagged in the screenshot I added to the PR.

File: Makefile [hunk 1]
```diff
@@ -81,7 +81,7 @@ test-with-cover:
 
 .PHONY: addlicense
 addlicense:
-	$(ADDLICENCESE) -y '' -c 'OpenTelemetry Authors' $(ALL_SRC)
+	$(ADDLICENCESE) -c 'The OpenTelemetry Authors' $(ALL_SRC)
```
Comment (tigrannajaryan) [inline Makefile:84]: Do we need "The"?

**Comment by tigrannajaryan** [comment]
@bruno-garcia please fix the merge conflict.

File: Makefile (hunk 1 above)
Comment (bruno-garcia) [reply Makefile:84]: Just trying to be consistent with the other repos but I'd be happy to drop it.

**Comment by bruno-garcia** [comment]
@tigrannajaryan fixed. Thanks

**Comment by bruno-garcia** [comment]
CI is failing on `loadtests`:

With:
`Error: No such image: cimg/go:1.14`

I'll try to trigger it again by reopening this PR

**Comment by bruno-garcia** [comment]
That didn't do it.
I can't re-run it on CircleCI because:
image

Can anyone bail me out here please?

**Comment by jrcamp** [comment]
I trigger it by doing a commit --amend to my previous change and force push.

**Comment by bruno-garcia** [comment]
CI is happy now.

**Comment by tigrannajaryan** [comment]
@bruno-garcia can you rebase or otherwise make a change to the commit to trigger a build again? For some reason codecov is stuck.

**Comment by bruno-garcia** [comment]
@tigrannajaryan I've pulled master into the branch and did another pass. Mainly hostmetricsreceiver that was added and included year.

**Comment by bruno-garcia** [comment]
@tigrannajaryan CI seems to be happy all the way now, including codecov \o/

**Pull request merged**